prog/kubernetes/provision_kubernetes_node.rb

```diff
@@ -131,4 +131,37 @@ def write_hosts_file_if_needed(ip = nil)
 
     nap 65536
   end
+
+  label def install_cni
+    script = <<BASH_SCRIPT
+#!/bin/bash
+cd /home/ubi || {
+  echo "Failed to change directory to /home/ubi" >&2
+  exit 1
+}
+exec ./kubernetes/bin/ubicni
+BASH_SCRIPT
+
+    cni_config = <<CONFIG
+{
+  "cniVersion": "1.0.0",
+  "name": "ubicni-network",
+  "type": "ubicni",
+  "ranges":{
+      "subnet_ipv6": "#{vm.ephemeral_net6.network}/80",
+      "subnet_ula_ipv6": "#{vm.nics.first.private_ipv6}",
+      "subnet_ipv4": "#{vm.nics.first.private_ipv4}"
+  }
+}
+CONFIG
+
+    sshable = vm.sshable
+    sshable.cmd("sudo tee /opt/cni/bin/ubicni", stdin: script)
+    sshable.cmd("sudo tee /etc/cni/net.d/ubicni-config.json", stdin: cni_config)
+
+    sshable.cmd("sudo chmod +x /opt/cni/bin/ubicni")
+    sshable.cmd("sudo iptables -t nat -A POSTROUTING -s #{vm.nics.first.private_ipv4} -o ens3 -j MASQUERADE")
```

Comment:
can you please find nftables version of this config for the sake of consistency.